examples/with-context-api/pages/_app.js

```diff
@@ -0,0 +1,19 @@
+import App, { Container } from 'next/app'
+/* First we import our provider */
+import NoteProvider from '../components/CounterProvider'
```

Comment:
@wesbos example used `NoteProvider`
But this here should be called `CounterProvider` now, right?

Reply:
YEs - sorry I ripped this out of another app :)

Reply:
Yes. it should be CounterProvider. I based on @wesbos example and forget to change that import